en/docs/fragments/artifact-binding-settings.md

```diff
@@ -0,0 +1,83 @@
+## Configure artifact expiration time
+
```

Comment:
Shall we add why a user needs to configure SAML 2.0 artifact binding in a couple of lines? FYI : Usually in SAML, the authentication request and assertion data is sent through the browser using POST or http REDIRECT binding. In case if you do not want to expose the entire message to the browser, then artifact binding can be used.

Reply:
ack..added this to the use-artifact-binding.md guide